src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -1064,7 +1096,15 @@ internal enum IncomingRequestType : byte
                 /// <summary>A request to cancel a request previously submitted to the agent.</summary>
                 Cancel,
                 /// <summary>A request to unpause the connection associated with a request previously submitted to the agent.</summary>
-                Unpause
+                Unpause,
+                /// <summary>A request to shutdown the agent and all active operations.  No easy request is associated with this type.</summary>
+                Shutdown
+            }
+
+            /// <summary>Exception thrown when the handler has been disposed of while there are active operations.</summary>
+            internal sealed class HandlerShutdownException : OperationCanceledException
```

Comment:
Oops, it was meant to be here:
https://github.com/dotnet/corefx/pull/7947/files/a9b9c5a88790cb3e2865b645fb57702591b91e6f#diff-e5ffb568f6f994c4243d35ef1b6830ceR524